membership-attribute-service/app/loghandling/Logstash.scala

```diff
@@ -1,20 +1,24 @@
 package loghandling
 
-import com.amazonaws.auth.AWSCredentialsProvider
 import com.amazonaws.util.EC2MetadataUtils
 import configuration.Config
-import play.api.Configuration
-import com.gu.aws.CredentialsProvider
+import com.gu.aws.ProfileName
 import com.typesafe.scalalogging.StrictLogging
+import software.amazon.awssdk.auth.credentials.{AwsCredentialsProvider, AwsCredentialsProviderChain, InstanceProfileCredentialsProvider, ProfileCredentialsProvider}
 
 case class LogStashConf(enabled: Boolean,
   stream: String,
   region: String,
-  awsCredentialsProvider: AWSCredentialsProvider,
+  awsCredentialsProvider: AwsCredentialsProvider,
   customFields: Map[String, String])
 
 object Logstash extends StrictLogging {
 
+  private val CredentialsProvider = AwsCredentialsProviderChain.builder
```

Comment:
we were previously getting this from the `membership-common` library.
But `kinesis-logback-appender` now expects the new aws sdk, so we need to create it differently.
Elsewhere we're still using the old aws sdk, but I don't want to start migrating that yet

Reply:
yes, this would be good to get upgraaded as for one thing it would reduce the jackson databind issues, but I agree not in this PR!